src/lwc/gsResources/gsResources.js-meta.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<LightningComponentBundle xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>49.0</apiVersion>
+    <isExposed>true</isExposed>
+    <masterLabel>Gs Resources</masterLabel>
```

Comment:
If "gs" stand for "Getting Started", maybe the label can be "NPSP Getting Started Resources"